kubernetes/setup.sh

```diff
@@ -15,7 +17,7 @@ for i in `seq 1 "${NUM_OF_NODES}"`;
 do
   cat ./kubernetes/test-node-base.yaml | sed "s/test-nodeX/test-node${i}/" > ".tmp/test-node${i}.yml"
   echo $i
-  echo "test-node${i}:/usr/local/openjdk-11/bin/java -Dmultinode.port=5000 -Dmultinode.udp-port=6000" >> ${DEST_HOST_FILE}
+  echo "test-node${i}:/usr/local/openjdk-11/bin/java -Dmultinode.port=$PORT" >> ${DEST_HOST_FILE}
```

Comment:
this is not going to work for standard nightly multi-node tests, in those we need to set up both Aeron TCP and UDP port.

Also, be aware that TCP and UDP ports need to be mapped accordingly in the yaml configuration:
https://github.com/akka/akka/blob/bd7105d47d1a67b5401facbc38a08b7efe89cda7/kubernetes/test-node-base.yaml#L53

Reply:
@andreaTP, have you seen the discussion [here](https://github.com/akka/akka/pull/30706#discussion_r718418163). I think this what you are referring to, correct? 

Some of the tests running on multi-node TCP, still uses UDP. Therefore we still need to have `-Dmultinode.udp-port=6000` for those tests. 

The annoying part here is that when running aeron-udp tests explicitly, we need to set multinode.port to 6000. This is the port that will be picked by the tests, not `multinode.udp-port`.

Anyway, I will look again into it. I would like to find a solution that is not a mix of conditions and configurations. Or at least something that we can easily follow.

Reply:
If it helps we could disable the AeronStream* tests when the artery transport is != aeron-udp. In that way we are only using Aeron (udp) in this new job, and not in the other.

Reply:
@octonato I thought we identified the issues and decided to move forward fixing this configuration:
https://github.com/akka/akka/blob/d703a2afe0e0c6dc8f1c0321b0f4652600e2ae99/akka-multi-node-testkit/src/main/scala/akka/remote/testkit/MultiNodeSpec.scala#L225

```
if udpPort.isDefined && transport == "aeron-udp" { ... use udp port }
```

Reply:
That won't work, you can't read he aeron-udp property before we load the configs. 😄  
That's why the `multinode` ones are passed through System properties. 

I changed the PR. There are three tests that runs with TCP configure, but streaming with aeron-udp. Therefore I changed this property to be explicit for those three tests. In all other case, we should use the selfPort either pointing to 5000 or to 6000. 

Not sure if that will make things more obvious. 

Please, have a look and let me know if it's easier to follow now.

Reply:
sounds good

Reply:
@octonato I find this approach more tricky to follow.

When merging the other multi-node tests we changed the code (where it was actually breaking) to handle the situation e.g.:
https://github.com/akka/akka/pull/30570/files#diff-d1f21345a0e5936bc83f960ae74727c574611d3a123bba7bbfb1623dd86a9fbcR55

check `MultiNodeSpec.configureNextPortIfFixed`, I would have preferred that approach, but not fussy about it.

Reply:
I don't understand how is that related with the problem here. The issue we have is that if the transport is aeron-udp, we should use 6000 when running in k8s. Otherwise, we use 5000. That should now be solved. 

That other property, `multinode.aeron-tests.port`, is only used for three specific tests that are not really testing through akka remote but are testing streams with aeron directly and therefore needs to use a udp port. 

Anyway, we can catch-up tomorrow about it.